ota-plus-web/app/com/advancedtelematic/api/ApiClient.scala

```diff
@@ -285,25 +301,24 @@ class DirectorApi(val conf: Configuration, val apiExec: ApiClientExec) extends O
       }
   }
 
-  def createRepo(namespace: Namespace, keyType: KeyType)(implicit ec: ExecutionContext): Future[Unit] = {
-    val body = Json.obj("keyType" -> TufCodecs.keyTypeEncoder(keyType).noSpaces)
```

Comment:
Can `Json.parse(keyType.json)` give unquoted value?

Reply:
`Json.parse(keyType.asJson.noSpaces)` does, as does `TufCodecs.keyTypeEncoder(keyType).asString.get`, but they both seem pretty obscure to me.